cmd/cmd-mewn.go

```diff
@@ -5,7 +5,7 @@ package cmd
b52021e7a21e8fc5dee2b1b77afc2d1e9724246bd8e1a0a1b96b5ba2d7d810f0d015e206a64b86f60c5c1dfe11ed1d449fa08c464995311ffe0d76119860a1af48b3d5b50594a755173087960962944c30be332121440bc234032e73ee92d99a4332c1770e242e5142546a92ca984f47e62819d08aad2303800777fd6e32ad3236862d475f91ca583d1e7b436032389162c701aa290631b5e43bf5c4be9ead817f1221f42d388c01588b1ec61f7fff7e9ad0818747638ddd18eb3d6c413221c81217ffcafcbb1e8f89e733d142978b03e48a2f9e218ca8ad012214710350d3b64e30071afada05fe7ae9630c948f770e5c22a25d063d84907d89c2faf7781fdae797116173009a6801b65a9a888d440bb0104b4719078ae38440fcf01ca18f2822f889a825ba31411d2c05b4c1a1ce6f47a8e3025fbb146907a846c0195a3b3e40614c3c81c480af9db01d4d1ecf3eb3b55df4f8625c0f967c9f0ffa3101ad50ea84b3a0bb44098d6c84350601396c538566eff69e1dedc7a6391de944eb71bfd1eef89e133d095246a378ece4519fb35146a3a3f58c38760336d1fe32828a580f268f4dfe4820750662ec1c3ca1e307e28f4d37faeaf7d8f50fa9a2edff4f09dfbf43885ed36a0dee6e0426a8dc764598feb40d4dc4fd28f6b922875709e8abb84e16ce8296086f6264d81e3b47e3849e617bc3776a0630fece8182c8940e7b58217f66daea32e11bbd2ffb8668f0996c8368f0ff295b940711d1e19b1ed872702936d3846eec251c813ccc87ef7b82d5b39ccbf430dffecc52d20ca8e6d39e328d2f5ed63411f298ea1287236e1f825eda5f1d82656f8021c211992326cea621078b6281838b0843dc5d433a8216d6b491cf2b19b0bc44aecf23d051b83bfb594af2f7f89e1d633fcb480bbcc1e7c4625a098219fa1a9e47ac9c258cb3d0c67cff3a1d13f9559cc03aac3858907e7d44fa5d81fe1c25a3c789b1741032fdd813ac9c780738870686a9660a20a96a5b84eae8e4c327f76051abf33728c1dae7e6268aa784727025717fe14ba500968f6d57aca82660b7230ebcc1e647312e1a3efc1d3f352f9baaa6cd846f078a378f0efe1b5b0f05de617337187fb2f38370128d25ef113f017d0a3a3f027dd2f717583723912b308f33c07f2f5affdfa2642cf67118ac9f5ed7262f4e008cdd9db08770e5e0daa1b60836e0ee503e93b1184a6a30c76b93a30d36d77011b8be6260aa03e20661c1fca122b230a128fa263a78491d0d02bf84f25d6e62c08fa6788a6ca18f09a7fb39ac01ae21f92faf003cf404b3913ffe90319b0cef7d507f69b8b603f468b5cac9de7fb9115cd716073f1df61051623148336e3b1dcb028d600e0faeed60dd6ff192a15621c9c72b8eebb7a1cde04a8f0bbf6998b1d8a1d89b63934130f5e6ff7a1301edf2a3aaeafc4f0d07137ce761a1539ff788220189f4135a920788e8d476012d18c25413dd43d8db04e131c63cc255e3d869a635ccf1bf32ff3e720b8a2f93cc5ceceed055b8f91ffe9f2518a08497f05fa69fb25494519a99448564f622480c825fcff6c29ed45d534e2c10394f74c9a05d860b73dbf5fdc205aced900f7eb8d93f41f8d79083fff9de831de77dc24eb7fc9707e93e4d06556e2d9092b8443f78c4a7c12938fd502892082ffe13ef39a875b4bacf58f66c4ee63b91b1b0e39844f5cd2df113ab3a703e931ad2bff976140c3e1378ef1f3bbcc861007a286fe6025ed68f37c3c93cd4871def97fed6fd5efa65cb074c1b848a5efdc86e16b0f31b9860097d451fb20c3988237c09b9d4d017943aecbbf8e1c25fd33df042d2e481ea4795146c9a7ed43e2aa59e943a80ddc41cff54b3779295c3254519bc131e2685da0fdbef6d3ddd12012c9998d025b53d7a7c45236a34e4820a64035a2dfaf6e6e8a9e77551d036cd767e852fe08dd4e0e9743afe69caf377a7d369e1639873b082ea35248c4713e25bfc5d1b968c5affa1133bf4b3812d8a0690f0a3a011de80bf7ad8f971a373f0f5af47ba7af498614f183beb05cff5eb08a967cef94e639fdcb2b5b50969d83ab6cb453dfdcddf88deeaff1b0000ffff2d8ed2993a1f0000")
```

Comment:
Any ideas why bridge.js has changed?

Reply:
hmmm something funny happened with branches. since it was a small PR I didn't mind, since my plan was to delete and fork again.

maybe I picked up a different version of bridge somehow in this kerfuffle

I'll try to fix it tho

Reply:
are u sure this is not supposed to happen? runtime seems altered too